### PR TITLE
Add course show action

### PR DESCRIPTION
**Pull request by chkhanch**

### CODE REVIEW

File: config/locales/en.yml [hunk 1]
```diff
@@ -51,3 +51,7 @@ en:
     world_history: 'World history'
     world_literature: 'World literature'
     writing: 'Writing'
+  activerecord:
+    attributes:
+      course:
+        user: Teacher
```
Comment (BohdanChaban) [inline config/locales/en.yml]: add newline

File: spec/controllers/courses_controller_spec.rb [hunk 2]
```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe CoursesController, type: :controller do
```
Comment (BohdanChaban) [inline spec/controllers/courses_controller_spec.rb]: add tests for request

File: app/controllers/courses_controller.rb [hunk 3]
```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+class CoursesController < ApplicationController
+  def show
+    @course = Course.find params[:id]
```
Comment (BohdanChaban) [inline app/controllers/courses_controller.rb]: you should use `includes` for group, subject and teacher against `find_records` method
http://guides.rubyonrails.org/active_record_querying.html#eager-loading-associations

and then refactor show view with these changes

File: app/controllers/courses_controller.rb [hunk 4]
```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+class CoursesController < ApplicationController
+  def show
+    @course = Course.includes('group', 'subject', 'user').where(id: params[:id])[0]
```
Comment (BohdanChaban) [inline app/controllers/courses_controller.rb]: where(id: params[:id])[0] == find(params[:id])

File: config/locales/ua.yml [hunk 5]
```diff
@@ -139,7 +118,7 @@ ua:
         role: Роль
         approved: Підтвердженний
         encrypted_password: "Зашифрований пароль"
-        failed_attempts: "Хибних спроб"
+        failed_attempts: "Хибниз спроб"
```
Comment (BohdanChaban) [inline config/locales/ua.yml]: This commit 
https://github.com/BohdanChaban/school/pull/38/commits/b7daa8529cc273f657a9ce085866ae5d622b6009
make some strange changes, remove translations. Is it ok?

Comment (chkhanch) [reply config/locales/ua.yml]: it`s wrong rollback :(

File: app/controllers/courses_controller.rb [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+    @course = Course.includes('group', 'subject', 'user').find(params[:id])
+    redirect_to root_path if redirect?
+  end
+
+  private
+
+  def redirect?
+    return true unless @course.displayed
+    return true if current_user.student? && @course.group_id != current_user.group_id
+    false
```
Comment (BohdanChaban) [inline app/controllers/courses_controller.rb]:
```
def redirect?
  !@course.displayed || @course.student_access_forbidden?
end
```

In Course model:
```
def student_access_forbidden?
    current_user.student? && group_id != current_user.group_id
end
```

**Pull request merged**